CHANGELOG.md

```diff
@@ -1,3 +1,26 @@
+# v70.0.0 (_2021-02-04_)
+
+[Full Changelog](https://github.com/mozilla/application-services/compare/v69.0.0...v70.0.0)
+
+## General
+
+### What's Changed
+
+- The bundled version of Glean has been updated to v34.0.0.
+- The bundled version of the Nimbus SDK has been updated to v0.7.2.
+
+## Autofill
+
+### ⚠️ Breaking changes ⚠️
+
+- The `NewCreditCardFields` record is now called `UpdatableCreditCardFields`.
+- The `NewAddressFields` record is now called `UpdatableAddressFields`.
```

Comment:
There's actually another change I should have mentioned in the last review - `update_address` and `update_credit_card` have had their signature changed - both now take the GUID to update as a param. I don't think it matters in practice though given our zero consumers.

Reply:
Ah, thanks. I agree the details of breaking changes here probably aren't too important in practice yet.